api/node/index.ts

```diff
@@ -412,23 +423,41 @@ export function loadFile(filePath: string, options?: LoadFileOptions): Object {
 
             let componentHandle = new Component(instance!);
             instance!.definition().properties.forEach((prop) => {
-                Object.defineProperty(componentHandle, prop.name.replace(/-/g, '_'), {
-                    get() { return instance!.getProperty(prop.name); },
-                    set(value) { instance!.setProperty(prop.name, value); },
-                    enumerable: true
-                })
+                let propName = prop.name.replace(/-/g, "_");
+
+                if (componentHandle[propName] !== undefined) {
```

Comment:
Minor nitpick, but this lookup will go through the prototype chain. I think it would be cleaner to use [hasOwn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) or [hasOwnProperty](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty).

Reply:
@tronical I tried to use `hasOWn` and `hasOwnProperty` before but both didn't work. Both always returned `false`.

Reply:
I will merge this now. If we found a better solution or you have something in mind how to get it work I can integrate it in the next PR.